@xen-orchestra/fs/src/s3.js

```diff
@@ -386,15 +386,6 @@ export default class S3Handler extends RemoteHandlerAbstract {
     } while (NextContinuationToken !== undefined)
   }
 
-  async _write() {
-    // patching a file on s3 if full of pitfalls
-    // between the patches smaller than the minimum size
-    // the patches bigger than 5GB
-    // encrypted file/blocks can't be patched
-    // we'll implement if it's really needed
-    throw new Error('not implemented')
-  }
```

Comment:
Merge with previous commit.

I'll merge this PR without squashing as it contains unrelated changes.